pipeline_steps/common.groovy

```diff
@@ -278,13 +278,54 @@ def archive_artifacts(){
   } finally{
     // still worth trying to archiveArtifacts even if some part of
     // artifact collection failed.
-    archiveArtifacts(
-      allowEmptyArchive: true,
-      artifacts: "artifacts_${env.BUILD_TAG}.tar.bz2,artifact_index.txt"
```

Comment:
The index is so that we can see what files are in the tar without having to download it. With the old archive artifacts method you could view artifact_index.txt without having to download it, which isn't the case anymore, so I didn't include it. I could try putting the contents of artifacts_index.txt into index.html so we keep that functionality.